### ISSUE TITLE
Keyboard shortcuts for tiling not working

### ISSUE THREAD

**Issue by shlozm** [bbidulock/icewm#647]
As per the man pages for icewm-keys which suggests the following:

These are key bindings for single window tile operations to replace
       the KeyWinArrange key bindings from the preferences file:

           key "Ctrl+Alt+KP_7" icesh -f sizeto 49% 49% top left
           key "Ctrl+Alt+KP_8" icesh -f sizeto 100% 49% top left
           key "Ctrl+Alt+KP_9" icesh -f sizeto 49% 49% top right
           key "Ctrl+Alt+KP_6" icesh -f sizeto 49% 100% top right
           key "Ctrl+Alt+KP_3" icesh -f sizeto 49% 49% bottom right
           key "Ctrl+Alt+KP_2" icesh -f sizeto 100% 49% bottom left
           key "Ctrl+Alt+KP_1" icesh -f sizeto 49% 49% bottom right
           key "Ctrl+Alt+KP_4" icesh -f sizeto 49% 100% top left
           key "Ctrl+Alt+KP_5" icesh -f sizeto 49% 49% center

I added the above to my keys file. I of course changed the keywinarrange shortcuts in my preferences to "", I also tried the above using Super as a modifier, nothing works. The only thing that works is using the arrow keys instead of the KP keys, such as key "Super+Left" icesh -f sizeto 49% 100% top left. (The keywinarrange keys do work when I don't have those keys assigned to tile operations)
What am I missing?

**Comment by cheapy** [comment]
I suspect those keys don't work on your keyboard.  Try assigning them to these keys instead:
key “Ctrl+Alt+KP_Home” icesh -f sizeto 49% 49% top left
key “Ctrl+Alt+KP_Up” icesh -f sizeto 100% 49% top left
key “Ctrl+Alt+KP_Prior” icesh -f sizeto 49% 49% top right
key “Ctrl+Alt+KP_Right” icesh -f sizeto 49% 100% top right
key “Ctrl+Alt+KP_Next” icesh -f sizeto 49% 49% bottom right
key “Ctrl+Alt+KP_Down” icesh -f sizeto 100% 49% bottom left
key “Ctrl+Alt+KP_End” icesh -f sizeto 49% 49% bottom left
key “Ctrl+Alt+KP_Left” icesh -f sizeto 49% 100% top left
key “Ctrl+Alt+KP_Begin” icesh -f sizeto 49% 49% center

PS: I got that from a post by blur13 on www.antixforum.com

**Comment by gijsbers** [comment]
Don't you have a keypad numlock?
Check what `xev` says for your KP.
Also consult xmodmap.

**Comment by shlozm** [comment]
@cheapy, I will try that thanks. @gijsbers, I tried it with numlock on and off,  xev does recognize the KP keys correctly, and the predefined Ctrl+Alt+KP_1 etc. to winarrange does work when not reassigned as I did. When I ran xmodmap I got this:


xmodmap:  up to 4 keys per modifier, (keycodes in parentheses):

shift       Shift_L (0x32),  Shift_R (0x3e)
lock        Caps_Lock (0x42)
control     Control_L (0x25),  Control_R (0x69)
mod1        Alt_L (0x40),  Alt_R (0x6c),  Meta_L (0xcd)
mod2        Num_Lock (0x4d)
mod3
mod4        Super_L (0x85),  Super_R (0x86),  Super_L (0xce),  Hyper_L (0xcf)
mod5        ISO_Level3_Shift (0x5c),  Mode_switch (0xcb)

I don't know what that signifies if anything...

Thanks for everything you do for IceWM!
